frontend/src/common/enums/SocketEvent.ts

```diff
@@ -0,0 +1,8 @@
+export enum SocketEvent {
```

Comment:
Alles super aber ich finde eher, dass enums jetzt nicht unbedingt in einen Ordner gehören, sondern dahin wo sie eingesetzt werden. Aber ist ok